src/styles/globals.css

```diff
@@ -1,3 +1,28 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import url("https://fonts.googleapis.com/css2?family=Rubik&display=swap");
```

Comment:
Here I already have the font set up in the tailwind config file, so this is not needed